python/foglamp/services/core/api/package_log.py

```diff
@@ -0,0 +1,73 @@
+# -*- coding: utf-8 -*-
+
+# FOGLAMP_BEGIN
+# See: http://foglamp.readthedocs.io/
+# FOGLAMP_END
+
+import os
+from pathlib import Path
+from aiohttp import web
+
+from foglamp.common.common import _FOGLAMP_ROOT, _FOGLAMP_DATA
+
+
+__author__ = "Ashish Jabble"
+__copyright__ = "Copyright (c) 2019, Dianomic Systems Inc."
+__license__ = "Apache 2.0"
+__version__ = "${VERSION}"
+
+_help = """
+    -----------------------------------------------------------
+    | GET            | /foglamp/package/log                   |
+    | GET            | /foglamp/package/log/{name}            |
+    -----------------------------------------------------------
+"""
+valid_extension = '.log'
+
+
+async def get_logs(request: web.Request) -> web.Response:
+    """ GET list of package logs
+
+    :Example:
+        curl -sX GET http://localhost:8081/foglamp/package/log
+    """
+    # Get logs directory path
+    logs_root_dir = _get_logs_dir()
+    found_files = []
+
+    for root, dirs, files in os.walk(logs_root_dir):
+        found_files = [f for f in files if f.endswith(valid_extension)]
```

Comment:
Please return a list of objects with { timestamp, plugin/service name  or "update", logFile }